### DISCUSSION TITLE
Welcome to circuitikz Discussions!

### DISCUSSION THREAD

**Discussion by Rmano** [circuitikz/circuitikz#813] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  Use this thread if you feel like introducing yourself (not needed!) or commenting on the general status of Discussions. If you have specific queries, ideas, suggestions, open a new Discussion!

**Comment by YalePhysicsLabs** [comment]
Hello! I am looking to draw a voltage regulator like a 7805 or LM3167 with a Vin, Vout and a `Vreg` and I haven't been able to find anything on the internets or in the manual. A simple rectangle with nodes for Vin, and out on the left and right respectively and a Vreg on the bottom. I would appreciate any help and apologies in advance if this subject has been addressed here already. Cheers!   -Stephen.

**Reply by YalePhysicsLabs** [reply]
I am the instructional lab director at Yale University and have been using circuitikz to draw my circuits for my lab handouts. Very powerful, but a steep learning curve. Thankfully, I haven't had to do anything too complicated.

**Reply by Rmano** [reply]
Moving to a new discussion #850
